static/js/lib/auth.js

```diff
@@ -58,3 +60,8 @@ export const isAnonAccessiblePath = (pathname: string): boolean =>
 
 export const needsAuthedSite = (): boolean =>
   !SETTINGS.allow_anonymous && !SETTINGS.authenticated_site.session_url
+
+export const goToFirstLoginStep = R.curry((history: Object, e: Event) => {
+  e.preventDefault()
+  history.push(LOGIN_URL)
+})
```

Comment:
could we use the `preventDefaultAndInvoke` wrapper with this function instead?

Reply:
I thought about that initially but forgot to try to implement. I'll take a look